app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_objects = {
+        person["name"]: Person(person["name"], person["age"])
+        for person in people
+    }
```

Comment:
1. Why is it dict, not list that should be returned?
2. You have Person.people dict for this purpose